source/community/release_notes/palm.rst

```diff
@@ -15,30 +15,220 @@ These are the release notes for the Palm release, the 16th community release of
+
+RateXBlock unsupported
+======================
+
+The RateXBlock has been deprecated and is now officially unsupported. `FeedbackXBlock 
+<https://github.com/pmitros/FeedbackXBlock>`_ is the recommended replacement.
+
+Stripe API changes
+==================
+
+Ecommerce now supports the new Stripe Payment Intents API and no longer uses the Stripe Charges API. The legacy 
+Stripe Charges ecommerce payment processor with a frontend in the ecommerce application is upgraded to a Stripe 
+Payment Intents payment processor with a frontend in frontend-app-payment. See the discussion post, `Ecommerce: 
+Changes to Stripe payment processor 
+<https://discuss.openedx.org/t/ecommerce-changes-to-stripe-payment-processor/9457>`_ for more details. 
+
+Ecommerce is still officially deprecated and we continue to work on the `Ecommerce Replacement Project 
+<https://openedx.atlassian.net/wiki/spaces/AC/pages/3617849345/Ecommerce+Deprecation+and+Replacement+Project>`_.
+
+
+Developer changes
+=================
+
+The old developer documentation site has been deprecated and removed. Please use `Open edX Developers - Latest 
+Documentation <https://docs.openedx.org/en/latest/developers/index.html>`_  instead.
+
+The modules :code:`tutor.hooks.actions`, :code:`tutor.hooks.filters`, and :code:`tutor.hooks.contexts` are no longer 
+part of the Tutor API. This change should not affect most developers, who only use the :code:`Actions` and 
+:code:`Filters` classes (notice the plural) from :code:`tutor.hooks`. If it does affect you, there are upgrade 
+instructions in the discussion post, `Simplifying & documenting the Tutor Hooks API 
+<https://discuss.openedx.org/t/simplifying-documenting-the-tutor-hooks-api/9258>`_
+
+Most of the viewing code for PDF certificates was removed in previous releases. In the Palm release, we closed out 
+this work with an ADR (Architecture Decision Record) on `Leaving PDF Certificate Fields in Certificates Model 
+<https://github.com/openedx/edx-platform/blob/open-release/palm.master/lms/djangoapps/certificates/docs/decisions/008-certificate-model-remnants.rst>`_. 
+
 
 Learner Experiences
 *******************
 
+Discussions Improvements
+========================
+
+* The Posts page has been streamlined, allowing users to see more information at once.
+* Post load faster 
+* Comments and responses can now be sorted in reverse order.
+* Filter by discussion with no responses. 
+
+Android App updates
+===================
+
+* The Learner dashboard was updated. Profile was moved from the header to the footer. Learners can choose between viewing My Courses and My Programs. 
+* The course header and course navigation have been updated
+* Learners will see a course dates shift banner in the course header when the server returns that dates needs shifting.  
+
+iOS App updates
+===============
+
+* Learners will see a course dates shift banner in the course header when the server returns that dates needs shifting.  
```

Comment:
I have no idea what this means, could you explain?

Reply:
Does the screenshot help?

Reply:
I don't think I see a screenshot from the iOS app

Reply:
whoops, forgot the `git push`